lib/jnpr/junos/device.py

```diff
@@ -218,7 +218,6 @@ def _sshconf_lkup(self):
                 self._hostname = found.get('hostname', self._hostname)
                 self._port = found.get('port', self._port)
                 self._conf_auth_user = found.get('user')
-                self._conf_ssh_private_key_file = found.get('identityfile')
```

Comment:
The information is still present in the SSH configuration that is provided to ncclient. So, your use case still work.

The line is removed because when `_conf_ssh_private_key_file` is defined, it disables the SSH agent (which is quite surprising as no other SSH client does that). So, if the key is stored in the agent, that doesn't work.